syntax = "proto3";

package contracts.services.documents.tariffs;

import "contracts/services/documents/tariffs/tariff.proto";

message CreateTariffRequest {
  string account_id = 1;
  bytes content = 2;
}

message CreateTariffResponse {
  Tariff tariff = 1;
}